frontend/styles/var/colors.ts

```diff
@@ -13,7 +13,7 @@ export const COLORS = {
   "ui-fleet-black-50": "#8B8FA2",
   "ui-fleet-black-33": "#B3B6C1",
   "ui-fleet-black-25": "#C5C7D1",
-  "ui-fleet-black-15": "#E2E4EA",
+  "ui-fleet-black-10": "#E2E4EA",
```

Comment:
this is the same hexadecimal as line 14 of colors.scss but they have different names

Reply:
🤔 I do not see the hex code anywhere else. Line 14 is `#B3B6C1`. Referencing the [style guide colors page](https://www.figma.com/file/qbjRu8jf01BzEfdcge1dgu/Fleet-style-guide-2022-(WIP)?node-id=210%3A11267&t=pduJSyKx2QjQUOep-0), it looks like there is no black-15, only black-10, so this change appears correct to me. Maybe I'm misunderstanding?

Reply:
I think @RachelElysia is referring to line 14 of frontend/styles/var/colors.scss, line 14 of which reads `$ui-fleet-blue-15: #e2e4ea;`, the same color as line 16 here (but named 10 instead of 15)